tests/SoCreate.ServiceFabric.PubSub.Tests/GivenServiceReference.cs

```diff
@@ -54,5 +54,246 @@ public void WhenDeterminingShouldDeliverMessageToActorWithUnmatchingPayload_Then
             bool shouldDeliver = actorRef.ShouldDeliverMessage(messageWrapper);
             Assert.IsFalse(shouldDeliver);
         }
+        [TestMethod]
+        public void WhenDeterminingShouldDeliverMessageToActorWithUnmatchingPayload_ThenReturnsTrue()
```

Comment:
Test name should be "WhenDeterminingShouldDeliverMessageToActorWithMatchingPayload_ThenReturnsTrue".